### PR TITLE
[ECMS-6711] upgraded ITHIT integration

### PR DESCRIPTION
**Pull request by MrHanachoo**

### CODE REVIEW

File: apps/resources-wcm/src/main/webapp/javascript/eXo/wcm/frontoffice/public/OpenDocumentInOffice.js [hunk 1]
```diff
@@ -29,35 +29,26 @@
 
   var uiActionBarContainer="";
 
+OpenDocumentInOffice.prototype.errorCallback = function (message) {
+        var installerFilePath = "/open-document/js/Plugins/" + ITHit.WebDAV.Client.DocManager.GetInstallFileName();
+
+        if (confirm("Opening this type of file requires a protocol installation. Select OK to download the protocol installer.")){
```
Comment (thomasdelhomenie) [inline apps/resources-wcm/src/main/webapp/javascript/eXo/wcm/frontoffice/public/OpenDocumentInOffice.js]: Do we really need a confirm here ? I mean, it is another click to do, and the user could probably cancel the installation at the first step of the installer, no ?

File: apps/resources-wcm/src/main/webapp/javascript/eXo/wcm/frontoffice/public/OpenDocumentInOffice.js [hunk 2]
```diff
@@ -29,35 +29,26 @@
...
+            window.open(installerFilePath);
+        }
+    }
```
Comment (thomasdelhomenie) [inline apps/resources-wcm/src/main/webapp/javascript/eXo/wcm/frontoffice/public/OpenDocumentInOffice.js]: Indentation is not good.

Comment (MrHanachoo) [inline apps/resources-wcm/src/main/webapp/javascript/eXo/wcm/frontoffice/public/OpenDocumentInOffice.js]: I will fix it

**Comment by MrHanachoo** [comment]
@thomasdelhomenie PR updated

**Comment by MrHanachoo** [comment]
replaced by #217 as this PR's branch has conflicts with exodev/develop.

**Pull request closed without merging**